README.rst

```diff
@@ -63,7 +63,7 @@ The documentation has these sections:
   programmatically with PyConcept EV.
 - `Examples <https://conceptev.docs.pyansys.com/version/stable/examples/index.html>`_: Explore examples
```

Comment:
these other links are also broken and need to have the index.html removed from them